npm-shrinkwrap.json

```diff
@@ -446,51 +460,54 @@
     },
     "nodemailer-smtp-transport": {
       "version": "0.1.13",
-      "from": "nodemailer-smtp-transport@^0.1.12",
+      "from": "https://registry.npmjs.org/nodemailer-smtp-transport/-/nodemailer-smtp-transport-0.1.13.tgz",
       "resolved": "https://registry.npmjs.org/nodemailer-smtp-transport/-/nodemailer-smtp-transport-0.1.13.tgz",
       "dependencies": {
         "nodemailer-wellknown": {
           "version": "0.1.4",
-          "from": "nodemailer-wellknown@^0.1.1",
+          "from": "https://registry.npmjs.org/nodemailer-wellknown/-/nodemailer-wellknown-0.1.4.tgz",
```

Comment:
After a second look at this, `version` and `resolved` haven't changed, so it seems to be ok. NTC.